taxi/urls.py

```diff
@@ -1,9 +1,23 @@
 from django.urls import path
 
-from .views import index
+from taxi.views import (index,
+                        ManufacturerListView,
+                        DriverListView,
+                        CarListView,
+                        CarDetailView,
+                        DriverDetailView)
 
 urlpatterns = [
     path("", index, name="index"),
+    path("manufacturer/",
+         ManufacturerListView.as_view(),
+         name="manufacturer-list"),
+    path("driver/", DriverListView.as_view(), name="driver-list"),
```

Comment:
```suggestion
    path("drivers/", DriverListView.as_view(), name="driver-list"),
```
Fix in all places must be plural